bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeperAdmin.java

```diff
@@ -1643,4 +1643,24 @@ private static boolean areEntriesOfSegmentStoredInTheBookie(LedgerMetadata ledge
         }
         return firstStoredEntryId != LedgerHandle.INVALID_ENTRY_ID;
     }
+
+    /**
+     * returns boolean value specifying if the ensemble of the segment is
+     * adhering to the ensemble placement policy for the given writeQuorumSize
+     * and ackQuorumSize.
+     *
+     * @param ensembleList
+     *            ensemble of the segment
```

Comment:
comment reads as if it is one segment. List of ensembles of the ledger or something